jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -819,7 +816,8 @@ public CommandLineBuilder getMainArgs(Set<String> parts) throws IOException
             {
                 for (Prop p : properties)
                 {
-                    cmd.addRawArg(CommandLineBuilder.quote(p.key) + "=" + CommandLineBuilder.quote(properties.expand(p.value)));
+                    if (!p.key.startsWith("java."))
```

Comment:
This broke `java.io.tmpdir`

Reply:
I'm not just excluding `java.version.`, which obviously we cannot set!